OpenDreamShared/Compiler/DM/DMParser.cs

```diff
@@ -216,6 +216,16 @@ public DMParser(DMLexer lexer, bool unimplementedWarnings) : base(lexer)
                     }
                 }
 
+                //TODO actual modified type support
+                if (Check(TokenType.DM_LeftCurlyBracket))
+                {
+                    if(_unimplementedWarnings) Warning("Modified types are currently not supported and modified values will be ignored.");
+
+                    while (Current().Type != TokenType.DM_RightCurlyBracket && !Check(TokenType.EndOfFile)) Advance();
```

Comment:
This could cause issues if there's a set of brackets inside the modified type. An example for a more correct way to parse this would be in the DMM parser [here](https://github.com/wixoaGit/OpenDream/blob/9b77863ea09be0dc3c72ec96e51af1c23b29ce1a/DMCompiler/Compiler/DMM/DMMParser.cs#L59).

Reply:
Excessive for what's essentially a stopgap until real modified types.